docs/content.zh/docs/dev/table/sql/create.md

```diff
@@ -186,6 +186,9 @@ CREATE TABLE [IF NOT EXISTS] [catalog_name.][db_name.]table_name
  | { INCLUDING | EXCLUDING | OVERWRITING } { GENERATED | OPTIONS | WATERMARKS } 
 }[, ...]
 
+<select_query>:
+使用来自SELECT语句的数据填充表。
```

Comment:
Ditto. In addition, please leave a space between Chinese and English characters.